lib/ansible/modules/network/netconf/netconf_config.py

```diff
@@ -262,6 +271,11 @@ def main():
             msg=module.params['datastore'] +
             ' datastore is not supported by this ansible module')
 
+    if module.params['save']:
+        if not ':startup' in m.server_capabilities:
```

Comment:
@wisotzky we need to change this to `if ':startup' not in m.server_capabilities:` to adhere to pep8